File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count={}
        count1={}
        if len(s)!=len(t):
            return False
        for k in s:
            if k in count.keys():
                count[k]+=1
            else:
                count[k]=1

        for m in t:
            if m in count1.keys():
                count1[m]+=1
            else:
                count1[m]=1

        if count==count1:
            return True
        else:
            return False

        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> set = new HashSet<>();
        for(int n:nums){
            set.add(n);
        }
        int longest = 0;
        for(int num:set){
            if(!set.contains(num-1)){
                int currentNum=num;
                int count = 1;
                while(set.contains(currentNum+1)){
                    currentNum++;
                    count++;
                }
                longest = Math.max(longest,count);
            }
        }
        return longest;
    }
}
